module/VuFind/src/VuFind/Export.php

```diff
@@ -385,4 +386,45 @@ public function getActiveFormats($context = 'record')
         }
         return $this->activeFormats[$context];
     }
+
+    /**
+     * Get the bulk export POST field name.
+     *
+     * @param string $format Format identifier
+     *
+     * @return string
+     */
+    public function getBulkExportPostField($format)
```

Comment:
Do you think we need BulkExport in the method names here given that this is in the context of an Export class, and these settings apply whether it's a single record being exported or a bulk batch? (I realize that at least one existing method may already be badly named based on these standards... just wondering if we should avoid propagating a mistake further, if indeed the existing naming is a mistake).

Reply:
I've cleaned up the method names. Thanks!